eng/Workarounds.props

```diff
@@ -42,4 +42,10 @@
     <ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles>
   </PropertyGroup>
 
+  <!-- Trim analyzers are automatically enabled by IsTrimming=true. They currently break when run on aspnetcore.
+       Remove when trimming analyzers are fixed: https://github.com/dotnet/aspnetcore/issues/41016 -->
+  <PropertyGroup>
+    <EnableTrimAnalyzer Condition=" '$(EnableTrimAnalyzer)' == ''">false</EnableTrimAnalyzer>
```

Comment:
I don't see this set anywhere else. Why include the `Condition`❔

Reply:
Just in case we want to enable for some projects later to test whether it works or not.

Reply:
/fyi the `Condition` doesn't help that case much because projects are read _after_ Directory.Build.props files and (therefore) this file. The project can therefore override any property it likes (as long as it doesn't use a `!= ''` `Condition` of course).

But my comment was definitely a nit.